### PR TITLE
Remove RO logical axioms

### PR DESCRIPTION
**Pull request by anitacaron**
Related to #2453

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224665,7 +224663,6 @@ xref: BFO:0000067
 id: continuous_with
 name: continuous with
 xref: RO:0002150
-is_symmetric: true
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224668]: From [ticket](https://github.com/obophenotype/uberon/issues/2453#issuecomment-1118435290):

> [Typedef]
> id: continuous_with
> name: continuous with
> xref: RO:0002150
> is_symmetric: true
>  This should be added to RO https://github.com/oborel/obo-relations/pull/610

Do not see instruction to delete this.

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -224996,7 +224993,6 @@ xref: RO:0002002
 id: has_branching_part
 name: has branching part
 xref: RO:0002569
-is_transitive: true
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224999]: From https://github.com/obophenotype/uberon/issues/2453#issuecomment-1118435290:

> [Typedef]
> id: has_branching_part
> name: has branching part
> xref: RO:0002569
> is_transitive: true
> Looks reasonable, although potential for massive proliferation of trivial inferences.
> 
>  Add to RO https://github.com/oborel/obo-relations/pull/612

Do not see instruction to delete this.

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (anitacaron) [reply src/ontology/uberon-edit.obo:224668]: I'm cleansing the logical axioms already in RO.

**Review by bvarner-ebi — APPROVED** [review]
Regarding 'has_skeleton', unsure from ticket if any action was desired (see comments below). Otherwise changes seem to be in line with associated issue.

> [Typedef]
> id: has_skeleton
> name: has skeleton
> xref: RO:0002551
> domain: UBERON:0000475 ! organism subdivision
> range: UBERON:0010912 ! subdivision of skeleton
> I think the domain maybe too strict. Doesn't my foot have a skeleton. Is a foot an organism subdivision?
> 
>  Get feedback from @cmungall
>  Remove axiom in UBERON? 

...
> has_skeleton domain: this may be a case of a domain trying to do the closed-world work of a DP. Although I am not sure it's particularly problematic either - foots/autopods are organism subdivisions

**Pull request merged**